### DISCUSSION TITLE
Will I install my old global npm packages one by one when I update the node version?

### DISCUSSION THREAD

**Discussion by remidosol** [coreybutler/nvm-windows#903] [General] [answered]
There is `reinstall-packages` command in nvm-sh package, does/will the nvm-windows support like this command?

Should I install my global dependencies one by one when I update the node version?
Thanks already.

**Comment by coreybutler** [comment, marked as answer]
Please see https://github.com/coreybutler/nvm-windows/wiki/Common-Issues#why-isnt-nvmrc-supported-why-arent-some-nvm-for-macoslinux-features-supported.
